stan/math/prim/scal/prob/exp_mod_normal_rng.hpp

```diff
@@ -18,9 +18,8 @@ namespace math {
  * given location, scale, and inverse scale using the specified random
  * number generator.
  *
- * mu, sigma, and lambda can each be a scalar, a std::vector, an
- * Eigen::Vector, or an Eigen::RowVector. Any non-scalar inputs must be the
- * same length.
+ * mu, sigma, and lambda can each be a scalar a vector. Any non-scalar
```

Comment:
missing " or "

Reply:
Thanks for catching that. I just pushed a fix.